### PR TITLE
Clean collection page css

### LINKED ISSUE #5348: Collection page cleanup
### Descriptive summary

General clean up on the Collection page

### Actual behavior

image

### Steps to reproduce the behavior

1. Do this
2. Then do this...

### Related work

Link to related tickets or prior related work here.

### PR DESCRIPTION
**Pull request by Dananji**
Fixes #5348

Changes proposed in this pull request:
* Override default `flex-wrap: wrap` injected from `.row` for the horizontal layout of the `.search-form` when screen width > 768px (there might be a better way to achieve this)
* Override `word-wrap: break-word` leaking from `div.card.collections-panel-wrapper` in `.search-form`

  Before: 
  wordbreak-bl7
  After:
  wordbreak-fix

* Replace `input-group-btn` class with `input-group-append`

  Before: 
  searchform-bl7
  After:
  searchform-fix

Guidance for testing, such as acceptance criteria or new user interface behaviors:
* Mainly look at collections and works page
* Check other pages where `search_form` is being used

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: app/views/catalog/_search_form.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <div class="input-group col-sm-9">
       <%= text_field_tag :q, current_search_parameters , class: "q form-control", id: "search-field-header", placeholder: t("hyrax.search.form.q.placeholder") %>
 
-      <div class="input-group-btn">
+      <div class="input-group-append">
```
Comment (adamjarling) [inline app/views/catalog/_search_form.html.erb:13]: @Dananji Looks like this is a nested `input-group`, which could be messing with things?  I wonder if we should approach this by looking at the markup surrounding the search forms themselves to see if they're aligned with BS4?

Comment (adamjarling) [reply app/views/catalog/_search_form.html.erb:13]: Maybe this route?  https://getbootstrap.com/docs/4.0/components/forms/#inline-forms

**Comment by Dananji** [comment]
Fixed the classes for the forms, references;
- https://getbootstrap.com/docs/4.0/components/forms/
- https://getbootstrap.com/docs/4.0/components/input-group/

**Review by adamjarling — APPROVED** [review]
Hey @Dananji This does look cleaner to me.  Thanks for looking into it!

**Pull request merged**